From 24a896269f9f3e86ac887f80c0c230dddbd8c61e Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Thu, 13 Aug 2015 07:23:07 -0700
Subject: [PATCH 228/827] remoteproc: zynqmp_r5_remoteproc: remove unused label

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Remove unused label "err_exit" from probe() function

Signed-off-by: Wendy Liang <jliang@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 670438284bcbf5aa77a1f18c995f59e6d07400c5)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/remoteproc/zynqmp_r5_remoteproc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/remoteproc/zynqmp_r5_remoteproc.c b/drivers/remoteproc/zynqmp_r5_remoteproc.c
index 5200498..cfb5a33 100644
--- a/drivers/remoteproc/zynqmp_r5_remoteproc.c
+++ b/drivers/remoteproc/zynqmp_r5_remoteproc.c
@@ -717,7 +717,6 @@ rproc_fault:
 dma_mask_fault:
 	dma_release_declared_memory(&pdev->dev);
 
-err_exit:
 	return 0;
 }
 
-- 
2.9.3

